src/ac_training_lab/picam/README.md

```diff
@@ -25,26 +25,40 @@ The following components are required for the equipment monitoring setup:
 - [MicroUSB Power Adapter (International Plugs)](https://www.pishop.ca/product/microusb-power-adapter-international-plugs/) - $12.95 CAD **[INTERNATIONAL OPTION]**
 - *Note: Pi Zero 2W uses micro USB for power, not USB-C*
 
+### Optional Troubleshooting Items
+These items are optional and don't factor into the final price shown. They're useful if you're unable to SSH into the machine:
+- [6FT High Speed HDMI Cable With Ethernet](https://www.pishop.ca/product/6ft-high-speed-hdmi-cable-with-ethernet-18gbps-28awg-gold-plated/) - $2.95 CAD **[OPTIONAL]**
+- [USB OTG Host Cable - MicroB OTG male to A female](https://www.pishop.ca/product/usb-otg-host-cable-microb-otg-male-to-a-female/) - $3.45 CAD **[OPTIONAL]**
+- [Wireless Keyboard and Mouse Combo with USB Dongle](https://a.co/d/dxV6WWj) - Amazon **[OPTIONAL]**
+- External Monitor **[OPTIONAL]**
+- *Note: These items provide a simple fix for situations where SSH access is unavailable*
+
 ### Mounting Hardware
 **Option A:** Off-the-shelf mount kit
 - [Pro Mini Camera Mount for Raspberry Pi Zero](https://www.pishop.ca/product/pro-mini-camera-mount-for-raspberry-pi-zero/) - $7.95 CAD
 - *Note: Off-the-shelf mount works but might need to be secured so the two laser cut pieces don't wobble relative to each other*
 
 **Option B:** DIY mounting solution (recommended):
 - [3D Printable PiCam Mount](https://github.com/AccelerationConsortium/ac-training-lab/tree/main/src/ac_training_lab/picam/_design) (STL files in repository)
-- [M2 Nylon Hex Standoff Spacer Screw Nut Assortment Kit (150 Pieces)](https://www.pishop.ca/product/m2-nylon-hex-standoff-spacer-screw-nut-assortment-kit-160-pieces/) - $7.95 CAD
-  - Alternative: M2 and M2.5 hardware from [McMaster-Carr](https://www.mcmaster.com/) as backup
+- **M2 Hardware (4 sets needed):**
```

Comment:
For all this M2 and M2.5 hardware, see if you can find nylon rather than metal, ideally in the smallest quantity available (a set would be fine if it's still cheap). If it's a pack that you're linking to, include that in the description, and make sure it's clear that 4 sets doesn't mean 4 of the packs, just that you can buy the pack and you only need 4 out of the X amount. Keep it concise but make sure it's clear 

I believe pishop has the M2.5 hardware needed, but not the M2 hardware (if I'm not mistaken).